PersonalBlog/urls.py

```diff
@@ -2,10 +2,18 @@
 from django.conf.urls.static import static
 from django.contrib import admin
 from django.urls import path,include
+from rest_framework import routers
+from BlogApp.views import UserViewSet, BlogPostViewSet
+
+router = routers.DefaultRouter()
+router.register(r'users', UserViewSet)
+router.register(r'blogposts', BlogPostViewSet)
```

Comment:
As i mentioned in my previous comment we need to remove these codes from Project/urls.py and add them to the App/urls.py. By the way if you pull my latest branch you will see that I have already implemented API, but I appreciate your effort. Thank you :)